packages/core-flows/src/definition/cart/workflows/update-line-item-in-cart.ts

```diff
@@ -32,6 +32,58 @@ export const updateLineItemInCartWorkflow = createWorkflow(
       data.input.item.variant_id!,
     ])
 
+    const salesChannelLocations = useRemoteQueryStep({
+      entry_point: "sales_channels",
+      fields: ["id", "name", "locations.id", "locations.name"],
+      variables: { id: input.cart.sales_channel_id },
+    })
+
+    const productVariantInventoryItems = useRemoteQueryStep({
+      entry_point: "product_variant_inventory_items",
+      fields: ["variant_id", "inventory_item_id", "required_quantity"],
+      variables: { variant_id: variantIds },
+    }).config({ name: "inventory-items" })
+
+    const confirmInventoryInput = transform(
```

Comment:
It seems this confirm inventory transformation is repeated in multiple steps, so we can probably extract it. Now I don't know what convention we want to follow with workflows and where we put this sort of "helper" code, but for the time being it can be part of the cart workflow folder?

Reply:
Agree, I am amid refactoring it. Will convert to draft in the mean time.